### DISCUSSION TITLE
modelling issue

### DISCUSSION THREAD

**Discussion by anujyadav0693** [neperfepx/neper#632] [Q&A]
sir,
i want to make a 2D cubic polycrystal model with random crystallographic orientation. which shows isotropic behaviour in macroscale. homogenised property should be isotropic.
QUES 1 what should i use -ori (uniform or random)?
QUES2 should i also use -oricrysym (cubic) for 2D model?

thank you.

**Comment by anujyadav0693** [comment]
sir,
To achieve this, the three reference Euler angles, which
define orientation, may be randomly distributed such that
their orientations lie equally spaced on the surface of a
sphere.
than what should i used -ori (uniform or random)?

**Reply by mkasemer** [reply]
Regarding question 1: Both types of orientation sets should roughly lead to isotropic behavior, given a high number of crystals. Your clarification in the second post, however, reveals that you are probably looking for a uniform orientation distribution, which will limit the variability in the ODF when compared to random (which has the potential for some minimal bias in the ODF).

Regarding question 2: Are your simulations truly 2D? That is, are the crystals planar, and you only need one angle to differentiate orientation? Or are you using the full 3D orientation? I don't think we know enough about your simulation setup or problem you wish to solve to give a proper answer.

**Reply by anujyadav0693** [reply]
Thanku for your valuable reply.
I am only giving one orientation angle about z axis.
If i am using cubic symmetry than all angles approximately preffer to certain directions

**Reply by rquey** [reply]
>  I am only giving one orientation angle about z axis.

In this case, use [-ori fiber(...)](https://neper.info/doc/neper_t.html?highlight=ori#cmdoption-ori).
